library/src/dotty/runtime/function/JProcedure10.java

```diff
@@ -8,7 +8,7 @@
 import scala.runtime.BoxedUnit;
 
 @FunctionalInterface
-public interface JProcedure10<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10> extends JFunction10<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, BoxedUnit>, java.io.Serializable {
+public interface JProcedure10<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10> extends scala.Function10<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, BoxedUnit>, java.io.Serializable {
     default void $init$() {
```

Comment:
I wonder if we could also remove all the empty $init$ methods, I don't think they serve any purpose.